itou/www/employee_record_views/forms.py

```diff
@@ -16,6 +16,42 @@
 from .enums import EmployeeRecordOrder
 
 
+class AddEmployeeRecordChooseEmployeeForm(forms.Form):
+    employee = forms.ChoiceField(
+        required=True,
+        label="Nom du salarié",
+        widget=Select2Widget,
+        help_text="Le salarié concerné par le transfert de données vers l'Extranet IAE 2.0 de l'ASP",
+    )
+
+    def __init__(self, employees, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.fields["employee"].choices = [(None, "Sélectionnez le salarié")] + list(
+            sorted(
+                [(user.id, user.get_full_name().title()) for user in employees if user.get_full_name()],
+                key=lambda u: u[1],
+            )
+        )
```

Comment:
```suggestion
        self.fields["employee"].choices = [(None, "Sélectionnez le salarié")] + sorted(
            [(user.id, user.get_full_name().title()) for user in employees if user.get_full_name()],
            key=lambda u: u[1],
        )
```
`sorted` renvoie déjà une liste.

Reply:
C'est pas faux... Je sais pas pourquoi j'avais en tête que ça renvoyais un objet `sorted` et donc on pouvais pas concaténer directement avec une liste.